dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -0,0 +1,2182 @@
+                                </VisualState>
+
+                            </VisualStateGroup>
+
+                        </VisualStateManager.VisualStateGroups>
+
+                        <Rectangle x:Name="ContentBackground" Fill="Transparent" />
+                        <TextBlock
+                x:Name="ExtensionGlyph"
+                Style="{StaticResource InkToolbarExtensionGlyphStyle}"
+                AutomationProperties.AccessibilityView="Raw" />
+                        <Grid RenderTransformOrigin="0.5, 0.5">
+                            <Grid.RenderTransform>
+                                <ScaleTransform x:Name="ContentTransform" />
+                            </Grid.RenderTransform>
+                            <TextBlock
+                  x:Name="Content"
+                  Style="{StaticResource InkToolbarGlyphFontStyle}"
+                  Text="&#xE76D;"
+                  AutomationProperties.AccessibilityView="Raw"
+                  Canvas.ZIndex="1"
+                  Foreground="{ThemeResource InkToolbarButtonForegroundThemeBrush}" />
+                            <TextBlock
+                  x:Name="CheckedContent"
+                  Style="{StaticResource InkToolbarGlyphFontStyle}"
+                  Text="&#xE76D;"
+                  AutomationProperties.AccessibilityView="Raw"
+                  Canvas.ZIndex="1"
+                  Foreground="{ThemeResource InkToolbarButtonSelectedForegroundThemeBrush}"
+                  Opacity="0" />
+                            <TextBlock
+                  x:Name="ContentFillGlyph"
+                  Text="&#xE88F;"
+                  Style="{StaticResource InkToolbarGlyphFillFontStyle}"
+                  TextAlignment="Center"
+                  AutomationProperties.AccessibilityView="Raw"
+                  Opacity="1"
+                  Canvas.ZIndex="0"
+                  Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=SelectedBrush, Mode=OneWay}" />
+                            <Border
+                  x:Name="Border"
+                  BorderBrush="{TemplateBinding BorderBrush}"
+                  BorderThickness="{TemplateBinding BorderThickness}"
+                  Grid.RowSpan="2" Grid.ColumnSpan="2"
+                  Margin="1,0"/>
+
+                        </Grid>
+                        <Rectangle x:Name="SelectionAccent" Style="{StaticResource InkToolbarSelectionAccentStyle}" />
+
+                    </Grid>
+
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+        <!--
+        The AttachedFlyout must be created from code behind, otherwise XAML will create *one* flyout for this style and
+        apply to all instances of this kind of InkToolbar button, which means all instances of this kind of InkToolbar
+        button share the same flyout.
+        Similarly, the Palette property also need to be created from code behind to avoid sharing.
+    -->
```

Comment:
Why is this comment here and not where we might expect the flyout or the top of this style? 

Also, nit: Indent "-->".

Also, do we need to include this comment in almost all styles?

Reply:
This comment is included in all styles, because they all contain flyouts getting created from code behind, moved these comments on top of each style